### DISCUSSION TITLE
Putting the screen to sleep only when music is not playing

### DISCUSSION THREAD

**Discussion by apastuszak** [rern/rAudio#538] [Feature Requests]
I have the official Raspberry pi touchscreen hooked up to my rAudio-1 pi, which I really like.

I'd like the screen to never go to sleep while music is playing, but go to sleep after 5 minutes when music is paused or my playlist ends. Is there any way to do that?

**Comment by rern** [comment]
Coming.

![browseronrpi](https://user-images.githubusercontent.com/10094236/138580066-9c46345c-ee77-4545-98f2-619ef22b97f3.jpg)
